test/jdk/ProblemList-fips.txt

```diff
@@ -150,6 +150,8 @@ security/infra/java/security/cert/CertPathValidator/certification/HaricaCA.java
 security/infra/java/security/cert/CertPathValidator/certification/LetsEncryptCA.java https://github.com/ibmruntimes/openj9-openjdk-jdk17/issues/116 linux-x64
 security/infra/java/security/cert/CertPathValidator/certification/SSLCA.java https://github.com/ibmruntimes/openj9-openjdk-jdk17/issues/116 linux-x64
 security/infra/java/security/cert/CertPathValidator/certification/TeliaSoneraCA.java https://github.com/ibmruntimes/openj9-openjdk-jdk17/issues/116 linux-x64
+sun/security/ssl/SSLSessionImpl/NoInvalidateSocketException.java
+sun/security/ssl/SSLSocketImpl/SSLSocketSSLEngineCloseInbound.java
```

Comment:
These lines don't follow the pattern of most others: three words (class, reason, platform): why not?

Reply:
updated